src/main/resources/application-perftest.properties

```diff
@@ -0,0 +1 @@
+datarepo.gcs.allowReuseExistingBuckets=true
```

Comment:
We've been defining spring profiles in the helm definition, so it would just get overwritten if we defined it here. But, we may want to reconsider where we are defining these. Thoughts? 

And yes, I think this is the desired profile name. It's specific to perf tests, not just the perf environment.

Reply:
I see it points to `perftest` in https://github.com/broadinstitute/datarepo-helm-definitions/pull/79 . 👍